README.markdown

```diff
@@ -34,11 +34,12 @@ Writing Objective-C? Check out our [Objective-C Style Guide](https://github.com/
 
 ## Naming
 
-Use descriptive names with camel case for classes, methods, variables, etc. Class names should be capitalized, while method names and variables should start with a lower case letter.
+Use descriptive names with camel case for classes, methods, variables, etc. Class names should be capitalized, while method names and variables should start with a lower case letter. Global constants, however, should have a project prefix.
 
 **Preferred:**
 
 ```swift
+let XYZSomeGlobalConstant = 123.456
```

Comment:
@Cananito Hahaa, nice experiment. Yes, that was along my understanding too. First and Second are the automatically generated modules and you do not need to do handle conflict between them.

I vote for `MyConstant` as well.

= You don't need to read the rest, but let me elaborate my argument for my own sake =
The stuff I was arguing by side tracking (global variable vs. name spacing) was conflict WITHIN the module, but that should be probably in another PR if needed.) 

In the following situation, `ViewAHeight` is declared as global constant so that it is accessible from outside of `ViewA`.  
```
let ViewAHeight = CGFloat(...)     // <=== Here

class ViewA: UIView {
    ...
}

class ViewControllerA: UIViewController {
    ...
    override func viewDidLoad() {
        super.viewDidLoad()
        ...
        view.height = ViewAHeight   // <== Here
        ...
    }
}    
```

But this can be written this way

```
class ViewA: UIView {
    static let height = CGFloat(...)     // <=== Here
    ...
}

class ViewControllerA: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        ...
        view.height = ViewA.height     // <=== Here
        ...
    }
}    
```

My preference as of now is to go with the latter, but I am not sure if we should enforce this in the style guide or not. Probably it is up to the developer.